cmd/registry/user.go

```diff
@@ -140,12 +170,22 @@ func initListUserCmd() {
 	userCreateCmd.MarkFlagRequired("parent")
 }
 
+func initReverseLookupUserCmd() {
+	flags := usersReverseLookupCmd.Flags()
+
+	flags.StringP("parent", "p", "", "Path to the parent org or group")
```

Comment:
to go along with the command design change as mentioned above, the parameter should be called `org` instead of `parent`

Reply:
The ID registry smart contract allows the user to build a hierarchical tree with as many layers as they want. Orgs should always be the top level and users should be the bottom. However, there can be levels between the two such as a group name ex.`/kaleido.io/engineering/developers/sam.stokes@consensys.net`. `parent` is a flexible term that refers to the next level up the ID registry tree. In this case `developers` is the parent of `sam.stokes@consensys.net`.